build/push-javadoc-to-gh-pages.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+if [ "$TRAVIS_PULL_REQUEST" == "false" ] && [ "$TRAVIS_BRANCH" ]; then
+
+  git config --global user.email "wps@us.ibm.com"
```

Comment:
Ummm, do we have to use a "watson"-related name here?  :)

Reply:
The dangers of copy and paste. I don't think I need those user lines anyway 😄